tools/kmod/sof_remove.sh

```diff
@@ -149,6 +149,7 @@ remove_module snd_soc_skl_hda_dsp
 remove_module snd_soc_sdw_rt700
 remove_module snd_soc_sdw_rt711_rt1308_rt715
 remove_module snd_soc_sof_sdw
+remove_module snd_soc_sof_es8336
```

Comment:
The code makes 8336 special but the commit message mentions 8326... typo in one or the other?

Either way, could you add a one-line comment so no one thinks the missing `insert_module` was a mistake?

Reply:
yeah it's confusing, I updated the commit message

Reply:
But no comment in the source?